deployment/hasura/metadata/databases/AerieMerlin/tables/public_plan_snapshot_activities.yaml

```diff
@@ -16,3 +16,26 @@ array_relationships:
         column_mapping:
           id: directive_id
           snapshot_id: snapshot_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [snapshot_id, id, name, source_scheduling_goal_id, created_at, last_modified_at, start_offset, type,
```

Comment:
Looks like this is still missing `last_modified_by`

Reply:
Huh, that should've been brought in in your PR. I'll update it 👍

Reply:
I guess this whole section of permissions didn't exist before, so it was easy to miss

Reply:
Ah, that explains it. Fix has been made.